library/globals.inc.php

```diff
@@ -2961,6 +2961,13 @@ function gblTimeZones()
             xl('Enable OpenEMR Patient Portal RESTful API.')
         ),
 
+        'rest_fhir_export_api' => array(
```

Comment:
like we discussed in call, would likely be more clear to have an Enable Client Credential grant with a recommendation of something like Turn on only if you know what you are doing :)

Reply:
So I dug into this more and I misspoke.  Any of the oauth2 grants can grant System permissions.  They can use password grant and a system admin user to get export privielges, they can use the authorization_code and get the system permissions.  What I think we should do is change this global to be 'Enable FHIR System Scopes (Turn on only if you know what you are doing).

This made me realize that I'm not checking the export scopes on each resource to make sure they have permission to access that resource.  I'll add that check in as we still need all of the System Scopes for the FHIR resources we have.

Reply:
I updated the global flag.  I also added to the FhirExportRestController a check to make sure the user has access to each resource scope in order to export the given resource.  

Found a couple bugs in AuthorizationController as part of this work that you may want to review my latest changes there.